### ISSUE TITLE
include honor certificates in grade freezing

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#2500]
Early runs of MicroMasters courses (both SCM and DEDP) offered honor certificates. The course teams have confirmed that would like to count these legacy certificates for MicroMasters credit.

**Comment by pdpinch** [comment]
@giocalitri what are our options for doing this? I've had fewer than 5 students request this kind of update, so if there's a manual way to do it that may be acceptable. But that number could increase when we get more SCM students coming to the portal.

**Comment by giocalitri** [comment]
I should have already done it in this way after the conversation we had about the old courses.
We need to test everything, of course.

**Comment by giocalitri** [comment]
Meaning this should be already in production

**Comment by pdpinch** [comment]
Ok. I'll assign this to myself to test then.

**Comment by pdpinch** [comment]
In my investigation, I haven't found any honor certificates amongst our cached certificates. Is it possible that we're not storing them from edX?

```
In [21]: from dashboard.models import CachedCertificate
    ...: s = set()
    ...: ct = set()
    ...: for cc in CachedCertificate.objects.all():
    ...:     s.add(cc.data['status'])
    ...:     ct.add(cc.data['certificate_type'])
    ...: print(s)
    ...: print(ct)
    ...: 
{'unverified', 'notpassing', 'downloadable', 'unavailable'}
{'verified'}
```

**Comment by giocalitri** [comment]
You are right: you can see [here](https://github.com/mitodl/micromasters/blob/master/dashboard/api_edx_cache.py#L241-L243) that we only cache the `verified` certificates.

Switching to all certificates can be tricky because there is a lot of logic built around the fact that the only passing certificates are the `verified` ones. [One example](https://github.com/mitodl/micromasters/blob/master/dashboard/utils.py#L197)
